spec/linter/v2/index-spec.js

```diff
@@ -0,0 +1,102 @@
+/* @flow */
+/* eslint-env jasmine */
+
+// import {
+//   typeDecl, exportedTypeDecl,
+//   func, exportedFunc,
+//   variables, exportedVariables,
+//   classDecl, exportedClassDecl,
+//   objectDestructuring, arrayDestructuring,
+//   exportedFuncWithObjectRest,
+// } from './parse-sample-ast'
+// import * as Parse from '../../lib/outline/parse'
+
+import { findCached, exec } from 'atom-linter'
+import * as LinterV2 from '../../../lib/linter/v2'
+
+declare var waitsForPromise
```

Comment:
Nit: Instead of defining this in each spec file you can define this in a `flow-typed/jasmine-atom.js` or a similar path